projects/igniteui-angular/src/lib/grids/column.component.ts

```diff
@@ -1768,25 +1793,67 @@ export class IgxColumnGroupComponent extends IgxColumnComponent implements After
     template: ``
 })
 export class IgxColumnLayoutComponent extends IgxColumnGroupComponent implements AfterContentInit {
+    public childrenVisibleIndexes = [];
     /**
      * Gets the width of the column layout.
      * ```typescript
      * let columnGroupWidth = this.columnGroup.width;
      * ```
      * @memberof IgxColumnGroupComponent
      */
-    get width() {
+    get width(): any {
```

Comment:
do you really need to change it from nothing to ```any``` ?

also, it seems you are parseInt-ing inside so it most likely can only return a number

Reply:
Base component defines type string for width. This was probably added to prevent incompatible type conversion. Otherwise it would resolve it as type number (due to the parseInt) and would throw an error.